cvelib/cve_api.py

```diff
@@ -112,13 +111,14 @@ def reserve(self, count: int, random: bool, year: str) -> Tuple[dict, str]:
         }
         if count > 1:
             params["batch_type"] = "nonsequential" if random else "sequential"
-        response = self._post("cve-id", params=params)
-        data = response.json()
-        return data, data["meta"]["remaining_quota"]
+        return self._post("cve-id", params=params).json()
```

Comment:
Might be keeping this on two lines, so you can easily tell whether ```response = self._post("cve-id", params=params)``` or ```return response.json()``` failed when looking at the traceback.

Reply:
I think that should be apparent from the type of the exception that this thrown, and I would also have to fix this for all of the other methods since they all use it :-) (maybe a future in misc-refactoring PR)